pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -273,6 +276,17 @@ type AlertmanagerConfigMatcherStrategy struct {
 	Type string `json:"type,omitempty"`
 }
 
+// AlertmanagerContinueStrategy defines the strategy used by AlertmanagerConfig objects to continue the alert routing.
+type AlertmanagerContinueStrategy struct {
+	// If set to `Always`, the operator will set to the first-level AlertmanagerConfig routes the value `continue: true`.
+	// If set to `ExceptLast`, the operator will set `continue: true` to all the routes and subroutes with the exception of the last one.
+	// `None` will not update the continue value specified in the AlertmanagerConfig.
+	// Default is `Always`.
+	// +kubebuilder:validation:Enum="Always";"ExceptLast";"None"
```

Comment:
I'm not convinced that we need a "None" strategy. At least I fail to see a use case.